scss/_base_button.scss

```diff
@@ -68,6 +71,20 @@
         margin-top: $spv-inter--condensed-scaleable + $spv-nudge-compensation;
       }
     }
+
+    &.is-small {
+      @extend %is-small;
+    }
+
+    &.is-v-condensed {
+      @extend %is-v-condensed;
+    }
+
+    th &,
+    td:not(.p-table-expanding__panel) & {
```

Comment:
Instead of having `:not` clauses, I think it makes more sense to introduce a new class that styles the expanding table button properly, e.g. `.p-table-expanding__button`.